src/lib/logger.ts

```diff
@@ -12,118 +23,65 @@ export const logger: Logger = pino({
     },
   },
   serializers: {
-    req: (req) => {
+    req: (req: Request) => {
       return {
         method: req.method,
-        url: req.url,
+        url: redactQueryParams(req.url),
```

Comment:
Happy to hear arguments against this. It avoids getting a log full of JAR requests from orchestration, and also things like authorization codes from CRI callbacks, but does mean we lose other bits of the query that could be useful to debug.

If there are specific 'useful' params, we might be able to add them separately (e.g. CRI?)

Reply:
I guess this will also affect error codes and description from CRI callbacks? definitely want to keep those!
and yeah CRI id would be useful